frontend/packages/pipelines-plugin/src/components/pipelines/modals/common/__tests__/utils.spec.ts

```diff
@@ -239,3 +240,36 @@ describe('convertPipelineToModalData', () => {
     expect(workspaces.filter((workspace) => workspace.type === 'PVC')).toHaveLength(3);
   });
 });
+
+describe('getPipelineName', () => {
```

Comment:
Initial look, you're missing a couple tests here:

- what if nothing is passed?
- what if nulls are passed for one / both?
- what if both pipeline AND pipeline run are passed?

Remember, Unit Tests are to catch all uses of a function, not just the ones from the expected path.

Reply:
added tests for following scenarios
- nothing being passed to the function, i.e. both values falsy
- both pipeline and pipeline run being passed to the function

added a check in the function to return null if both arguments are falsy. So do we need another test for null values, since it will treat null as falsy value?